give.php

```diff
@@ -515,6 +518,27 @@ private function setupExceptionHandler()
         $handler = new UncaughtExceptionLogger();
         $handler->setupExceptionHandler();
     }
+
+    protected function disableVisualDonationFormBuilderFeaturePlugin()
+    {
+        // Include plugin.php to use is_plugin_active() below.
+        include_once ABSPATH . 'wp-admin/includes/plugin.php';
+
+        // Prevent fatal error due to a renamed class.
+        class_alias(TestOffsiteGateway::class, 'Give\PaymentGateways\Gateways\TestGateway\TestGatewayOffsite', true);
+
+        if ( is_plugin_active('givewp-next-gen/give-visual-form-builder.php' )) {
+            deactivate_plugins(['givewp-next-gen/give-visual-form-builder.php']);
+
+            add_action('admin_notices', function() {
+                Give()->notices->register_notice([
+                    'id' => 'give-visual-donation-form-builder-feature-plugin-deactivated',
+                    'description' => __('The Visual Form Builder Beta plugin is no longer needed, since that functionality has been included in the version of GiveWP you are using. To prevent conflicts, the Beta plugin has been deactivated. You can safely delete that plugin.', 'give'),
```

Comment:
```suggestion
                    'description' => __('The Visual Form Builder Beta plugin is no longer needed, since the form builder is included in your current version of GiveWP. To prevent conflicts, the Beta plugin has been deactivated and can be safely deleted.', 'give'),
```